app/actions/index.js

```diff
@@ -166,16 +176,19 @@ const updateConnections = dat => dispatch => {
 
 const updateState = dat => {
   const key = encode(dat.key)
-  const state = !dat.network
+  const state = pausedDats[dat.key]
     ? 'paused'
-    : dat.writable || dat.progress === 1
-      ? 'complete'
-      : dat.network.connected ? 'loading' : 'stale'
+    : !dat.network
+      ? 'disconnected'
```

Comment:
if a dat doesn't have a network, it's paused. Why add an extra duplicate state?

Reply:
https://github.com/dat-land/dat-desktop/pull/507#issuecomment-363664012

Reply:
Why would you close the connection? In dat node, `dat.joinNetwork()` will create `dat.network` immediately and it will only be removed if you call `dat.leaveNetwork()`

Reply:
If joinNetwork fails it was still open, had an issue with this.

Reply:
Interesting, I haven't experienced that yet. Shouldn't dat-node throw in this case?